idflakies-legacy/src/main/java/edu/illinois/cs/dt/tools/detection/LegacyDetectorPathManager.java

```diff
@@ -97,6 +103,39 @@ public Path cachePathInstance() {
         }
     }
 
+    @Override
+    public Path startsPathInstance() {
+        String outputPath = Configuration.config().properties().getProperty("dt.incdetector.startspath", "");
+        Logger.getGlobal().log(Level.FINE, "Accessing startspath: " + outputPath);
+        if (outputPath == "") {
+            return modulePath().resolve(".starts");
+        } else {
+            Path outputPathObj = Paths.get(outputPath);
```

Comment:
Maybe these should be relative paths (so use `resolve`), in case of multi-module projects (same for Ekstazi stuff).